components/Tabs/docs/theming-extensions.md

```diff
@@ -0,0 +1,40 @@
+### Theming Extensions (Beta)
+
+ `MDCTabBar` supports Material Theming using a Container Scheme. The code is currently
+in Beta readiness, which means you will need to follow the [instructions for adding the
+MaterialComponentsBeta podspec to your
+project](https://github.com/material-components/material-components-ios/blob/73bdc03c2bd2abd032b0b69f05cd76928361aa37/contributing/beta_components.md#beta-program-for-components).
+There are two variants for Material Theming of a MDCTabBar, which are the Primary Variant
+color and the Surface Variant.
+
+ <!--<div class="material-code-render" markdown="1">-->
+
+ #### Swift
+
+ ```swift
+// Import the TabBar Theming Extensions module
+import MaterialComponentsBeta.MaterialTabs_MaterialTheming
+ ...
+ // Initialize your app's Container Scheme
+let containerScheme = MDCContainerScheme()
+ // Apply the Container Scheme to either Primary Theme
```

Comment:
This should be something like, "Theme the tab bar with either the Primary Theme"  "or the Surface Theme".  The Container Scheme isn't being applied to the Primary Theme.

Reply:
For this one, I am fine with the change, but these words (especially "Apply") are used a lot in many other components too. I think we should change them all or not for convention/document purpose. And it probably should happen in a separate PR.

Reply:
Let's please fix this new code and we can clean-up the others later. New code should not continue errors made in the past.

Reply:
Addressed in 014c401360d496476332f1822b707265928fec51